### DISCUSSION TITLE
各位绳匠七夕准备跟谁过呀

### DISCUSSION THREAD

**Discussion by Fishingxd** [share121/inter-knot#396] [其他]
不会是自己一个人过得吧？    

巧了不是，我也是（

**Comment by Nirvana341** [comment]
什么，七夕？这节日和我有什么关系，忽略了

**Reply by Fishingxd** [reply]
言之有理，令人信服

**Comment by WORLD-executeME** [comment]
一个人过呗，顶多去录像店借盘爱情片的录像带看看

**Comment by ShangHuaWenYuan** [comment]
当然是跟伊埃斯过啦嘿嘿嘿🤤🤤🤤香香甜甜的伊埃斯

**Comment by 666waht** [comment]
和绳网上认识的绳匠大半夜开拓一个叫提瓦特的空洞，最见鬼的是她是我暗恋对象

**Comment by Kamisato100** [comment]
不会有人七夕还蹲在家里打游戏吧，不会吧不会吧

什么，原来是我自己啊
![16CCB860](https://github.com/user-attachments/assets/3df98f46-bd57-4952-952e-f385eeb94818)

**Comment by Kamisato100** [comment]
为什么发个表情会变得这么大……不好意思各位，看来只能用emoji不能用搜狗输入法自带的表情
